### PR TITLE
[stable/metabase] Add tolerations and affinity

### PR DESCRIPTION
**Pull request by amit-o**
Signed-off-by: Amit Oren <amitoren224@gmail.com>

#### What this PR does / why we need it:
Add tolerations and affinity in addition to the node selector already present

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by amit-o** [comment]
davidkarlsen

File: stable/metabase/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: The easy, open source way for everyone in your company to ask questions
   and learn from data.
 name: metabase
-version: 0.7.0
+version: 0.7.1
```
Comment (davidkarlsen) [inline stable/metabase/Chart.yaml]:
```suggestion
version: 0.8.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by amit-o** [comment]
@davidkarlsen ping

**Review by cpanato — APPROVED** [review]
/lgtm

**Pull request merged**